easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -270,8 +326,51 @@ def configure_step(self):
             raise EasyBuildError("Failed to move non-espresso directories: %s", err)
 
     def install_step(self):
-        """Skip install step, since we're building in the install directory."""
-        pass
+        """Custom install step for Quantum ESPRESSO."""
+
+        # Copy all binaries
+        try:
+            # binary
+            bindir = os.path.join(self.installdir, 'bin')
+            shutil.copytree(os.path.join(self.cfg['start_dir'], 'bin'), bindir)
+
+            upftools = []
+            if 'upf' in self.cfg['buildopts'] or 'all' in self.cfg['buildopts']:
+                upftools = ["casino2upf.x", "cpmd2upf.x", "fhi2upf.x", "fpmd2upf.x", "ncpp2upf.x",
+                            "oldcp2upf.x", "read_upf_tofile.x", "rrkj2upf.x", "uspp2upf.x", "vdb2upf.x",
+                            "virtual.x"]
+                if LooseVersion(self.version) > LooseVersion("5"):
+                    upftools.extend(["interpolate.x", "upf2casino.x"])
+                if LooseVersion(self.version) >= LooseVersion("6.3"):
+                    upftools.extend(["fix_upf.x"])
+            upf_bins = [os.path.join('upftools', x) for x in upftools]
+            for x in upf_bins:
```

Comment:
please avoid using single-letter variable names outside of list comprehensions, use a more meaningful name like `upf_bin`